python/pyspark/serializers.py

```diff
@@ -199,6 +211,55 @@ def __repr__(self):
         return "ArrowSerializer"
 
 
+class ArrowPandasSerializer(ArrowSerializer):
+    """
+    Serializes Pandas.Series as Arrow data.
+    """
+
+    def __init__(self):
+        super(ArrowPandasSerializer, self).__init__()
```

Comment:
Do we need this?

Reply:
No, that was leftovers.. I'll remove it in a followup.